configure.ac

```diff
@@ -23,7 +23,7 @@ AC_CANONICAL_HOST
 dnl ##
 dnl ## Setup automake
 dnl ##
-AM_INIT_AUTOMAKE([1.11 -Wall foreign subdir-objects])
```

Comment:
These changes are already in the `master` branch, it looks like you'll need to rebase your branch on the master branch: `git fetch origin master && git rebase master` (`origin` might have to be `upstream` or something else, depending on your setup.